project/Dependencies.scala

```diff
@@ -61,7 +61,7 @@ object Dependencies {
   private val googleOauth2V = "1.1.0"
   private val googleOauthClientV = "1.32.1"
   private val googleCloudResourceManagerV = "1.1.2"
-  private val grpcV = "1.41.0"
+  private val grpcV = "1.45.0"
```

Comment:
Release notes were very detailed, and I searched Cromwell for references to anything noted to have changed.